matheus_massula_test/lib/pages/qr_code.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:matheus_massula_test/cubit/qr_code/cubit/qr_code_cubit.dart';
+import 'package:matheus_massula_test/resources/string_constant.dart';
+import 'package:matheus_massula_test/services/widgets/app_dependencies.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+
+import 'widgets/seconds_count_down.dart';
+
+class QRCodeContainer extends StatelessWidget {
+  const QRCodeContainer({ Key? key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final dependencies = AppDependencies.of(context);
+
+    return BlocProvider(
+      create: (BuildContext context) {
+        final cubit = QrCodeCubit(qrCodeWebClient: dependencies!.qrCodeWebClient);
+        cubit.getQRCode();
+        return cubit;
+      },
+      child: _QRCodePage(),
+    );
+  }
+}
+
+class _QRCodePage extends StatelessWidget {
+  const _QRCodePage({ Key? key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(StringConstant.QR_CODE_PAGE_TITLE),
+        centerTitle: true,
+      ),
+      body: BlocBuilder<QrCodeCubit, QrCodeState>(
+        builder: (context, state) {
+          if(state is QrCodeLoaded) {
+            return _buildLoadedBody(context, state);
+          } else if(state is QrCodeError) {
+            return _buildErrorBody(context, state);
+          } else {
+            return _buildLoadingBody();
+          }
+        },
+      ),
+    );
+  }
+
+  Container _buildLoadingBody() => Container(
+    height: 400,
+    width: 400,
+    child: Center(
+      child: CircularProgressIndicator()
+    )
+  );
+
+  Column _buildErrorBody(BuildContext context, QrCodeError state) => Column(
+    children: [
+      Container(
+        height: 400,
+        width: 400,
+        child: Center(
+          child: Icon(
+            Icons.error,
+            color: Theme.of(context).errorColor,
+          )
+        )
+      ),
```

Comment:
I didn't understood exactly what was the suggestion, but I would be happy to talk more about it.

Reply:
The changes you made resolved the issue. Unless you're going to be putting these widgets in a row, there isn't really any use for hardcoding the width of these since you want to align them to the center of the screen regardless.

Reply:
Sorry about that, my comment was related to your comment on the file `matheus_massula_test/lib/pages/qr_code.dart`